src/sophia/eventgen.f

```diff
@@ -263,6 +268,8 @@ subroutine eventgen(L0,E0,eps,theta,Imode)
        endif
 
 c... consider only stable particles:
+       if (REMDEC .eq. 0) then !anpros 2022/07/22: Leave decayed 
```

Comment:
Don't use inline comments in F77 code. There make random problems with some f2py versions: F77 = comment character, free form via !. No need to break the standard if not crucially needed.